.circleci/config.yml

```diff
@@ -16,8 +16,6 @@ jobs:
       - restore_cache:
           keys:
           - gnarails-{{ checksum "gnarails.gemspec" }}
-          # fallback to using the latest cache if no exact match is found
-          - gnarails-
```

Comment:
Why is this going away?

Reply:
#103 changed the cache key to be the gemspec file (part of the change here), as it removed the Gemfile.lock.

This change modifies the gemspec, so it breaks the cache. However, circle was falling back to the cache key `gnarails-`, which should also not be used, because it doesn't have...the things that are part of this change. For workflow jobs, there also apparently isn't a "Build without cache" option - or if there is, I can't find it.

Getting rid of this does make it less likely to have a cache hit, but I think that's acceptable due to 1) the number of times this should be modified, in reality 2) any changes that cause an initial cache miss probably won't want to rely on the fallback anyway.